File: app/src/main/java/com/odin/movielist/Utils/MoviesApi.java
package com.odin.movielist.Utils;
import com.odin.movielist.Models.Movies;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
/**
 * Created by mobil on 16.03.2017.
 */

public interface MoviesApi {
    @GET("/")
    Call<Movies> getMovies(
            @Query("s") String movieName,
            @Query("type") String movieType

    );
}

File: app/src/main/java/com/odin/movielist/utils/MoviesApi.kt
package com.odin.movielist.utils

import com.odin.movielist.models.Movies
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by Dino Omanovic on Mar 16, 2017
 */

interface MoviesApi {
    @GET("/")
    fun getMovies(
            @Query("s") movieName: String,
            @Query("type") movieType: String

    ): Call<Movies>
}
